### ISSUE TITLE
Why is “ALLSKY_EXTRA” on the SD card

### ISSUE THREAD

**Issue by Alex-developer** [AllskyTeam/allsky-modules#40]
### Discussed in https://github.com/AllskyTeam/allsky-modules/discussions/37

<div type='discussions-op-text'>

<sup>Originally posted by **Adler6907** December 17, 2023</sup>
Modules like `allsky_dewheater.py` constantly write to `/home/pi/allsky/config/overlay/extra` which is not in RAM like `/home/pi/allsky/tmp`.
Wouldn't it be better to change `ALLSKY_EXTRA="${ALLSKY_OVERLAY}/extra"` to `ALLSKY_EXTRA="${ALLSKY_TMP}/extra"` in `/home/pi/allsky/variables.sh`?

Then in the example the file `allskydew.json` in `/home/pi/allsky/tmp/extra` would be constantly written.
</div>

**Comment by EricClaeys** [comment]
Agree.  Anything that isn't needed across reboots should be in ALLSKY_TMP.
Who put "extra" directory where it currently is????  :-)
